sources/packages/web/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
+    
```

Comment:
Please check if this change is required.

Reply:
Done